docker-compose.yml

```diff
@@ -77,6 +77,8 @@ services:
     depends_on:
       store:
         condition: service_healthy
+#      loader:
+#        condition: service_completed_successfully
```

Comment:
Not needed, as they are not automatically started.

Reply:
It depends, if you never remove your volume it is unnecessary. If you do, your integration tests will fail when running `docker compose --profile test down --volumes && docker compose --profile test up -d`.

Reply:
The profile is not intended to be used together with `up`. Try this instead
`docker compose down --volumes && docker compose --profile test build && docker compose up -d && docker compose run --rm loader && docker compose run --rm integration`

(copied from what the github workflow does).